features/step_definitions/generator_steps.rb

```diff
@@ -28,6 +35,11 @@
   EOH
 end
 
+Given(/^I debug$/) do
```

Comment:
@jkeiser Omnibus isn't installed that way though...it's pulled in via a `bundle install` from an Omnibus project. Either way we should place `pry` in a `:development` group in the `Gemfile` OR probably just make it a development dependency in the gemspec:

```ruby
gem.add_development_dependency 'pry'
```